### PR TITLE
Add Web-Ticker@3.0.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by finch185277**
Pull request for issue: #10629

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/mazedigital/Web-Ticker
 * Official website (optional, not the repository):
 * NPM package url (optional):https://www.npmjs.com/package/webticker
 * License and its reference: GPLv3
 * GitHub / Bitbucket popularity (required):https://maze.digital
   - Count of watchers:19
   - Count of stars:150
   - Count of forks:86
 * NPM download stats (optional):
   - Downloads in the last day:10
   - Downloads in the last week:24
   - Downloads in the last month:108

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/Web-Ticker/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.webticker*"
+        ]
+      }
+    ]
+  },
+  "licenses": [
```
Comment (sashberd) [inline ajax/libs/Web-Ticker/package.json]: Lets leave only GPL-3.0

Comment (extend1994) [reply ajax/libs/Web-Ticker/package.json]: @sashberd cdnjs need commercial license if there is one. See https://github.com/cdnjs/cdnjs/pull/11551#issuecomment-322674874

**Comment by PeterDaveHello** [comment]
Duplicated with #12276?

**Comment by extend1994** [comment]
Yes, they are duplicated. Seems like #12276 happened to be opened after #12269, which has issues for intern4, had opened.
@finch185277 Please pay attention that you should open a pull request after making sure it's not opened by others. 
I think it would be better for you to close this pull request by yourself to remember this. Open another one when you close this one. Thanks!

**Comment by finch185277** [comment]
Yes

**Pull request closed without merging**